packages/gulp-bem-xjst/lib/gulp-bem-xjst.js

```diff
@@ -113,9 +113,10 @@ module.exports.toHtml = function(templatesStream) {
 
                 templatesVinyls.forEach(function(file) {
                     file.data || (file.data = _eval(String(file.contents)));
+                    var BEMHTML = file.data.BEMHTML || file.data[Object.keys(file.data)[0]];
```

Comment:
there's no access to opts here so we try default and fall back to any existing key as it's the only one exported at the moment